content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -262,11 +263,10 @@ well as lower-trust users.The following listed controls should be enforced/disal
 				The RuntimeDefault seccomp profile must be required, or allow specific additional profiles.<br>
 				<br><b>Restricted Fields:</b><br>
 				spec.securityContext.seccompProfile.type<br>
-				spec.containers[*].securityContext.seccompProfile<br>
-				spec.initContainers[*].securityContext.seccompProfile<br>
+				spec.containers[*].securityContext.seccompProfile.type<br>
+				spec.initContainers[*].securityContext.seccompProfile.type<br>
 				<br><b>Allowed Values:</b><br>
-				'runtime/default'<br>
-				undefined / nil<br>
```

Comment:
dropping undefined/nil here is tricky, since this can be specified at either the pod level or the container level or both.

If `spec.securityContext.seccompProfile.type=="RuntimeDefault"`, then it's ok for the container-level fields to be undefined, and vice-versa...